### PR TITLE
add file wordpress.yml

### PR DESCRIPTION
**Pull request by hectork01**

### CODE REVIEW

File: wordpress.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        TO wordpress@localhost
+        IDENTIFIED BY 'passwone';
+        FLUSH PRIVILEGES;
+        EOF
+        
+        #Running script to create the DB
+        cat wordpress.sql | sudo mysql --defaults-extra-file=/etc/mysql/debian.cnf
+        
+        #Retriving the Public IP address
+        IPVAR=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
```
Comment (szkl) [inline wordpress.yml]: ```ec2metadata --public-ipv4``` command would be a better way to get this info.

File: wordpress.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        
+        #Enabling the new site and Restarting the apache service
+        a2ensite wordpress
+        service apache2 restart
+        
+        #Writing the file to connect WP to the DB and Password definition
+        cat - <<EOF >> /etc/wordpress/config-localhost.php
+        <?php
+        define('DB_NAME', 'wordpress');
+        define('DB_USER', 'wordpress');
```
Comment (szkl) [inline wordpress.yml:47]: This can be a user input template variable.

File: wordpress.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        define('DB_PASSWORD', 'passwone');
```
Comment (szkl) [inline wordpress.yml:48]: This can be a user input template variable.

File: wordpress.yml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        define('DB_HOST', 'localhost');
```
Comment (szkl) [inline wordpress.yml:49]: This can be a user input template variable.

File: wordpress.yml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        EOF
+        
+        #Enabling the new site and Restarting the apache service
+        a2ensite wordpress
+        service apache2 restart
+        
+        #Writing the file to connect WP to the DB and Password definition
+        cat - <<EOF >> /etc/wordpress/config-localhost.php
+        <?php
+        define('DB_NAME', 'wordpress');
```
Comment (szkl) [inline wordpress.yml:46]: This can be a user input template variable.

File: wordpress.yml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        CREATE DATABASE wordpress;
+        GRANT SELECT,INSERT,UPDATE,DELETE,CREATE,DROP,ALTER
+        ON wordpress.*
+        TO wordpress@localhost
+        IDENTIFIED BY 'passwone';
+        FLUSH PRIVILEGES;
+        EOF
+        
+        #Running script to create the DB
+        cat wordpress.sql | sudo mysql --defaults-extra-file=/etc/mysql/debian.cnf
```
Comment (szkl) [inline wordpress.yml]: No need to create `wordpress.sql` file.

```shell
mysql --defaults-extra-file=/etc/mysql/debian.cnf <<EOF
CREATE DATABASE wordpress;
GRANT SELECT,INSERT,UPDATE,DELETE,CREATE,DROP,ALTER ON wordpress.*
TO wordpress@localhost
IDENTIFIED BY 'passwone';
FLUSH PRIVILEGES;
EOF
```

Comment (szkl) [reply wordpress.yml]: Don't forget to replace database info used here with user input variables.

**Review by szkl — CHANGES_REQUESTED** [review]

**Comment by hectork01** [comment]
@szkl The 2 changes requested are done; however, as we talked, the use of the variables as user input, will be an improvement when available.

**Review by szkl — APPROVED** [review]

**Pull request merged**